src/useCircularDrag.ts

```diff
@@ -1,23 +1,23 @@
-import { MouseEvent, TouchEvent, useEffect, RefObject, useState } from 'react'
+import { MouseEvent, TouchEvent, useEffect, RefObject, useState, useCallback } from 'react'
 import { useCircularInputContext } from './'
 
 export function useCircularDrag(ref: RefObject<SVGElement | null>) {
 	const { onChange, getValueFromPointerEvent } = useCircularInputContext()
 	const [isDragging, setDragging] = useState(false)
 
-	const handleStart = (e: MouseEvent | TouchEvent) => {
+	const handleStart = useCallback((e: MouseEvent | TouchEvent) => {
 		if (!onChange) return
 		stopEvent(e)
 		setDragging(true)
 		const nearestValue = getValueFromPointerEvent(e)
 		onChange(nearestValue)
-	}
+	}, [onChange])
```

Comment:
Now that we're adding a dependency list, we should make it exhaustive and include all of them just to be safe from unusual bugs.

I believe it's something like `[onChange, setDragging, stopEvent, getValueFromPointerEvent]`. In this case it doesn't change much as the references won't change but it's good practice.

https://reactjs.org/docs/hooks-reference.html#conditionally-firing-an-effect

Same goes for all the others (whichever way we choose to move forward)

What do you think?

Reply:
Hey, sorry for the delay, I'm just getting back from traveling. Yup, I think you're right and it should be exhaustive. Will be less likely to run into problems in the future. I'll make the change and get it pushed up.

Reply:
Amazing, thank you. Hope you enjoyed your travels!

Reply:
Thanks =). I added those additional deps in, let me know if there are any other changes you'd like to see!

Reply:
this is great!